plugins/frontend/backstage-plugin-github-insights/src/plugin.ts

```diff
@@ -97,3 +111,13 @@ export const EntityGithubInsightsEnvironmentsCard =
       },
     }),
   );
+
+export const GithubInsightsMarkdownContent = githubInsightsPlugin.provide(
```

Comment:
This is exported so it can be used by the markdown plugin when I get to that patch.